kahuna/public/js/upload/jobs/required-metadata-editor.html

```diff
@@ -65,14 +67,15 @@
                     gr-datalist-input
                     ng-model="ctrl.metadata.credit"
                     ng-change="ctrl.save()"
+                    ng-disabled="!ctrl.userCanEdit"
                     ng-model-options="{ updateOn: 'gr-datalist:update blur' }" />
             </gr-datalist>
 
             <button
                 class="job-editor__apply-to-all"
                 title="Apply this credit to all"
                 type="button"
-                ng-if="ctrl.withBatch"
+                ng-if="ctrl.withBatch && ctrl.userCanEdit"
```

Comment:
can we also use `ng-disabled` here?

Reply:
I guess we can, but if the ng-if is false I think it won't matter if it's disabled, is that right? or are you saying we replace the ng-if with ng-disabled? in that case, I think the button will still show, but clicking it won't do anything.

Reply:
sorry, my mistake